src/main/resources/application.properties

```diff
@@ -1,4 +1,7 @@
 spring.data.mongodb.uri=${SIC_CODE_API_MONGO_URL}
 spring.data.mongodb.database=${SIC_CODE_API_DATABASE}
-
 spring.data.mongodb.auto-index-creation=true
+# server.port has been added here because the api could not be accessed while in a docker container.
```

Comment:
Was there a port collision when using the default port (8080) ?

Reply:
When the port is set to 8080 here I'm getting a bad gateway error when running the api on dockerCHS

Reply:
I can see you've added an environment variable for this instead of hardcoding the value - I think that is a lot better as it'll provide environment specific configuration.